redisbp/example_monitored_client_test.go

```diff
@@ -5,93 +5,147 @@ import (
 
 	"github.com/go-redis/redis/v7"
 
+	"github.com/reddit/baseplate.go/log"
 	"github.com/reddit/baseplate.go/redisbp"
 	"github.com/reddit/baseplate.go/tracing"
 )
 
+// ExampleService is an example go-kit service to help demonstrate how to use
+// redisbp.type MonitoredCmdableFactory in a service.
+type ExampleService struct {
```

Comment:
Try use `My*` instead of `Example*`, except that one function, so that the godoc server will render the example correctly: https://pkg.go.dev/testing?tab=doc#hdr-Examples

>The entire test file is presented as the example when it contains a single example function, at least one other function, type, variable, or constant declaration, and no test or benchmark functions.

Reply:
Ya, `go vet` yelled at me so I changed it to just `Service`.

Reply:
Oh right.

Now you have more than one example functions in that file, so it won't be rendered as whole file in the doc site. Maybe just use one and explain that you can also use other options in comments?